contents/blog/open-source-business-models.md

```diff
@@ -0,0 +1,68 @@
+---
+date: 2021-04-30
+title: How We Monetized Our Open Source DevTool
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+By [Mo](https://twitter.com/shehuphd), Content Lead @ PostHog
+
+So, you’ve decided to build an open-source product.
+
+While your choice is highly commendable, the resounding applause from devs on GitHub and HackerNews won’t pay the bills. At some point, you’ll need to monetize your product to sustain its development (and your team) in the long run.
+
+Just one thing: **you need to build a community.**
+
+<iframe src="https://giphy.com/embed/OF0yOAufcWLfi" width="480" height="238" frameBorder="0" class="giphy-embed" allowFullScreen></iframe><p><a href="https://giphy.com/gifs/celebs-jimmy-fallon-OF0yOAufcWLfi">via GIPHY</a></p>
+
+Open-source requires a decently-sized, highly engaged community for any business model to work. People have to believe in your solution enough to contribute code, comments, or cash to it. If money is a priority for your business, open-source might not be the path for you.
+
+At PostHog, we’ve been fortunate enough to build an engaged community of contributors over the years, and we celebrate our top contributors with each new release. You can check out our [contributor page](https://posthog.com/contributors), see our [GitHub repo](http://github.com/PostHog/), and join our [Slack community](https://posthog.com/slack) of users.
+
+Let’s now look at how we’ve managed to turn a buck with PostHog over the past few months — plus 3 other business models to explore for your own product.	
+
+## Option #1: Support
+Support involves helping large companies deploy, use, maintain, and upgrade your software. These companies usually lack the time, expertise, or inclination to do all that by themselves, and as the developer, you’re in a better position to tailor the deployment to their specific situation, deploy updates as needed, and extract a lot of value.
+
+The downside is that support-based open source business models tend to return lower margins. This is fine if you’re bootstrapping, but trying to raise VC money becomes a [schlep](https://www.google.com/search?q=schlep) as VC firms generally prefer high-margin startups.
+
+At PH, we started with the support model before running into one problem: developers typically want to wrangle your software themselves before paying for help or maintenance. 
```

Comment:
Style thing - change to PostHog (we just use that everywhere - helps reinforce who we are for first time readers)

Reply:
Amended